bin/start-pgbouncer-stunnel

```diff
@@ -7,6 +7,11 @@ main() {
     exec "$@"
   fi
 
+  if is-enabled "${PGBOUNCER_MONITORING_ENABLED:-1}"; then
```

Comment:
here as well, we are really relying on the user and password being set as well but aren't testing for it